scripts/install

```diff
@@ -138,19 +137,24 @@ ynh_use_logrotate
 #=================================================
 ynh_script_progression --message="Modifying a config file..." --weight=3
 
-cp ../conf/config.json $final_path/config/config.json
+#cp ../conf/config.json $final_path/config/config.json
+
+user_pwd=$(ynh_string_random --length=24)
```

Comment:
I would suggest something like `smtp_user_pwd`, so we know which user we're writing about.